code/_onclick/cyborg.dm

```diff
@@ -14,7 +14,7 @@
 	if(check_click_intercept(params,A))
 		return
 
-	if(stat || lockcharge || IsParalyzed() || IsStun())
+	if(stat || (lockcharge && A.loc != src) || IsParalyzed() || IsStun())
```

Comment:
How would you be able to click on something whose `loc` is the robot?

Reply:
The borg tablet has a hud icon (which is clicked to open), but physically exists inside the borg. Basically I need borgs to be able to access the tablet at all times, because it now holds functions like stating laws. Bypassing the click and tgui lockdown restrictions for objects inside the borg seems like the least hacky way to do this.

Reply:
can't you handle the hud icon's click separately?
this is odd as a general check

Reply:
Fixed via vis_contents.